entrypoint.sh

```diff
@@ -81,7 +67,7 @@ restore() {
 
     local gpg_file="$ENCRYPTED_PATH/$1"
     if file_not_exists "$gpg_file"; then
-        if is_aws_s3_enabled; then
+        if ! s3sync is_enabled; then
```

Comment:
Isn't this backwards?  I would think if it's not enabled then you wouldn't attempt to sync.  Could do with an `else` for the scenario where it's not enabled.